### DISCUSSION TITLE
如何彻底删除

### DISCUSSION THREAD

**Discussion by thf0rth** [LTCatt/PCL2Help#309] [Q&A]
请问一下就是我在删除pcl2文件夹下所有文件之后为什么我再次安装后还是会有我的离线信息，还有就是我在启动器当中删除的游戏好像只是删除了游戏本体其他文件没有删除，如何解决这个问题

**Comment by Chiloven945** [comment]
显然这里是帮助库，不是寻求帮助的地方
需要寻求帮助请在PCL库的discussion进行提问
